src/Core/Checkout/Customer/SalesChannel/LoginRoute.php

```diff
@@ -120,6 +120,7 @@ public function login(RequestDataBag $data, SalesChannelContext $context): Conte
             [
                 'id' => $customer->getId(),
                 'lastLogin' => new \DateTimeImmutable(),
+                'languageId' => $context->getLanguageId(),
```

Comment:
Can you add a testcase in the `\Shopware\Core\Checkout\Test\Customer\SalesChannel\LoginRouteTest` that checks that  the languageId is updated as expected?

Reply:
@keulinho Added the testcase!